README.md

```diff
@@ -22,20 +22,28 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+## Getting started
+For an overview of Palisade, start with the Palisade introduction and the accompanying guides: QuickStart Guide; and Developer Guide: [Palisade README](https://github.com/gchq/Palisade).
+
 # Palisade Clients
 
 ## Client Implementations
+Implementation of a client will need to send requests into Palisade and process the response.  
+The workflow for this is expected to follow the pattern of first sending in a request to register access for data.  
+This will return a response consisting of a token (a unique id) that can then be used in a second request to retrieve the processed data.
+An implementation of a Client will need to conform to this workflow, but the technology that is used can be left to best suit the intended use of the application.
 
-The job of the client code is to send the request for data into Palisade and to interpret the result as required for the data processing technology it is written for.
-The responsibility for implementations of the client code is to provide users with a way to request data from Palisade in a way that the user has to make minimal changes to how they would normally use that processing technology.
-Implementations of this component will usually require deep understanding of the data processing technology in order to best hook into that technology, without needing to fork the code for that technology.
-
-To find out more information on our technology specific clients, please see the following modules:
-* [Akka Client](client-akka/README.md)
-* [Fuse Client](client-fuse/README.md)
+The following examples demonstrate the use of different kinds of clients that can operate with the Palisade service:
 * [Java Client](client-java/README.md)
+  Java based RESTFul client using an API similar to the JDBC  
 * [Shell Client](client-shell/README.md)
-
+  Command Line Interface(CLI) client which provides interactive operations.
+* [Fuse Client](client-fuse/README.md)
+  Filesystem in Userspace (FUSE) client which uses a CLI for interactive operations and stores the results in a FUSE filesystem.  
+* [Akka Client](client-akka/README.md)
+  Akka client which provides access an interface using both Java and Akka types.
+* [S3 Client](client-s3/README.md)
+  Akka client which provides access an interface using both Java and Akka types and data is prepared and then stored on an Amazon Web Service (AWS) Simple Cloud Storage (S3) bucket from where is it retrieved.
```

Comment:
am not sure this statement is correct. This client is a microservice that exposes an S3 API REST endpoint which technologies that can normally read data from S3 compliant API's can then route their S3 commands via this microservice to seamlessly get the data access protections palisade provides

Reply:
Took another stab at it.  It now reads "Akka Microservice which provides an endpoint for accessing resources stored in an Amazon Web Service (AWS) Simple Cloud Storage (S3) bucket.  
  The example uses [Apache Spark](https://sparkjava.com/) for RESTful queries with the microservice."